syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -277,13 +277,25 @@
 				{
 					"match":"(?i)\\b(CONCAT)\\b",
 					"name":"support.function.scl"
+				},
+				{
+					"match":"(?i)\\b(LOWER_BOUND)\\b",
+					"name":"support.function.scl"
+				},
+				{
+					"match":"(?i)\\b(UPPER_BOUND)\\b",
+					"name":"support.function.scl"
+				},
+				{
+					"match":"(?i)\\b(REF)\\b",
+					"name":"support.function.scl"
 				}
 			]
 		},
 		"constants":{
 			"patterns": [
 				{
-					"match": "\\b(t|T|TIME)#(((([-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?)|\\d+)?d)?((([-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?)|\\d+)?h)?((([-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?)|\\d+)?m)?((([-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?)|\\d+)?s)?((([-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?)|\\d+)?ms)?)\\b",	
+					"match": "\\b(S5TIME|S5T|TIME|LTIME|T|LT|D|TOD|LTOD|DT|DTL|LDT)#[0-9|+|-]*([^(\\s|;)]+)",	
```

Comment:
not sure about this. Maybe we should hava a seperate one for 
```
DT#1990-01-01-00:00:00.000
DT#2089-12-31-23:59:59.999
LDT#1970-01-01-00:00:00.000000000
LDT#2262-04-11-23:47:16.854775807
DTL#1970-01-01-00:00:00.0
DTL#2262-04-11-23:47:16.854775807
LTOD#00:00:00.000000000 
LTOD#23:59:59.999999999
TOD#00:00:00.000 
TOD#23:59:59.999
D#1990-01-01
D#2169-06-06
```
This solution does still highlight this `T#106751ns_23asdasdasgfah_47s_16m_854d_775us_808ns124214s` which to my knowledge is invalid timeformat (both order and characters).

Reply:
We should definately add nano and micro seconds!